medusa/server/api/v2/info.py

```diff
@@ -33,6 +71,32 @@ def get(self, query=''):
             'trimZero': app.TRIM_ZERO,
             'fanartBackground': app.FANART_BACKGROUND,
             'fanartBackgroundOpacity': app.FANART_BACKGROUND_OPACITY,
+            'branch': app.BRANCH,
+            'commitHash': app.CUR_COMMIT_HASH,
+            'release': app_version,
+            'sslVersion': ssl_version,
+            'pythonVersion': sys.version[:120],
+            'databaseVersion': {
+                'major': cur_branch_major_db_version,
+                'minor': cur_branch_minor_db_version
+            },
+            'os': platform.platform(),
+            'locale': '.'.join([str(loc) for loc in app_locale]),
```

Comment:
@ratoaq2 should we check in travis if user adds a str() function and make travis fail?